Basic_viewer/include/CGAL/Buffer_for_vao.h

```diff
@@ -899,16 +900,16 @@ class Buffer_for_vao
   std::vector<BufferType>* m_pos_buffer;
   std::vector<IndexType>* m_index_buffer;
   std::vector<BufferType>* m_color_buffer;
-  std::vector<BufferType>* m_flat_normal_buffer;
-  std::vector<BufferType>* m_gouraud_normal_buffer;
+  mutable std::vector<BufferType>* m_flat_normal_buffer;
+  mutable std::vector<BufferType>* m_gouraud_normal_buffer;
 
```

Comment:
Instead of have `mutable` in `Buffer_for_vao.h`, to be able to flip the normals, the fragment shader for facets could take an extra uniform parameter of type `float`, with value `+1` or `-1`. That would also avoid resyncing the buffers with the GPU: there would be just a change of the uniform variable.

Reply:
Previously discussed here: https://github.com/CGAL/cgal/pull/6967#discussion_r1363882189

Reply:
Maybe, please do the modification if you want.